Jason prodev updates

jmheaster opened the pull request.

jmheaster committed:
Create prodev.md

Beginning to split out prodev and community participation files

jmheaster committed:
Create community-participation.md

Beginning to split out prodev and community participation files

jmheaster committed:
Update community-participation.md

Copied content from the prodev-community-participation.md file before it was deleted

jmheaster committed:
Delete prodev-community-participation.md

Delete as part of splitting this into two separate files

jmheaster committed:
Update orientation-meetings.md

Fixed links to prodev and community participation, which are now separate files

jmheaster committed:
Update prodev.md

Major text updates & changes to prodev policy content

jmheaster committed:
Update prodev.md

Fixed link & minor formatting change

nerdstein commented on docs/03-policies/prodev.md:
```diff
@@ -0,0 +1,101 @@
+
+Historical Prodev Hours Spent: 
+
+| Year  | Company-wide Prodev Hours Used | # People Billing Prodev Hours | Average Prodev Hours Used Per Person Annually |
```
@jmheaster for historical purposes, it might be useful to show how many people total were in the company at the time and percentage

jmheaster replied:
@nerdstein I like the idea of using participation percentage as a KPI for prodev and will add it to the appendix section for later use (assuming we can find historical data on total people in the company in a given year).  I would not include that participation data in the above table, as it is meant to be a guide for people on how much prodev time is typically used by people.

nerdstein replied:
Cool

jmheaster committed:
Update prodev.md

Added text in appendix section for future KPI of percentage of prodev participation

nerdstein commented on docs/03-policies/prodev.md:
```diff
@@ -0,0 +1,102 @@
+
+CivicActions supports and encourages the Professional Development (prodev) of each team member by providing an annual budget of $1,200 per person to be used for prodev-related expenses.
+
+In addition to the $1,200 annual budget, each team member is encouraged to use a “take what you need” approach to using time for prodev pursuits during business hours. If you need guidance on what amount of time is appropriate to spend on prodev in a given year, consult the table below.
```
I wonder if there should be a statement in here about potential impact to billable work. If someone takes what they need and it negatively impacts a deliverable, I would hope this could be a red flag or would warrant someone pinging their PM upfront to request this time when it's least disruptive.

nerdstein commented on docs/03-policies/prodev.md:
```diff
@@ -0,0 +1,102 @@
+ 
+ In order to provide some additional structure, support and focus to the prodev efforts of individuals, 1 month long prodev sprints will be held on a regular basis. During these sprints, participants will attend scrum meetings 2 to 3 times per week to talk about their prodev progress being made and connect with others pursuing their prodev goals at the same time. The sprint will also consist of a ...
+
+The total time commitment for participating in a prodev sprint will vary widely depending on what topic and intensity level each participant chooses to pursue, but the prodev sprint is not considered a full time project.
```
Might this vary if someone is doing pro dev the company would like to sponsor? I could see a situation where someone submits a plan that bolsters our skills in a strategic area, it gets approved, and they use this sprint structure to "check in" on their efforts/learnings

nerdstein commented on docs/03-policies/prodev.md:
```diff
@@ -0,0 +1,102 @@
+
+The total time commitment for participating in a prodev sprint will vary widely depending on what topic and intensity level each participant chooses to pursue, but the prodev sprint is not considered a full time project.
+
+Assignment to a prodev sprint is voluntary, and team members are encouraged to express their desire to participate to the prodev Practice Lead, Jason Heaster. Efforts will be made to group people with similar prodev goals or certification requirements. Once assigned to a sprint, participants’ project assignment schedules will be adjusted in Forecast to allocate time for the prodev sprint during t ...
```
didn't realize this was becoming a practice. it makes sense to give it some focus, for sure.

nerdstein replied:
We could also ask people to join in groups if people are interested in learning something together. see: pattern lab -- iris/richard

nerdstein commented on docs/03-policies/prodev.md:
```diff
@@ -0,0 +1,102 @@
+
+## Asking a Mentor, Coach, or Peer to Help
+
+For some people, getting input from others is an important part of the professional development process. You are encouraged to seek out other CivicActioners who you think could help you achieve your prodev goals. This may be a manager, a peer, or any person who is good at something you’d like to learn.
```
I would also mention the opportunity of seeking expertise outside of the organization. I could see that being really beneficial.

nerdstein commented on docs/03-policies/prodev.md:
```diff
@@ -0,0 +1,102 @@
+A: Books, courses, conferences, certifications and prodev-related travel expenses.
+
+Q: Are there any other limits on the prodev budget?  
+A: For spending amounts over $50, you’ll need to get approval in advance. [Use this form](https://goo.gl/forms/4z1GQsBtrx6m4lVy2) to request approval. Anything under $50 can be purchased without permission as long as it supports professional development goals. Receipts and expense sheets should be submitted in the usual way, noting as professional development costs.
```
"the usual way" -- we should probably link to the handbook page that mentions Harvest (does this exist?)

nerdstein commented on docs/03-policies/prodev.md:
```diff
@@ -0,0 +1,102 @@
+* How do we measure prodev progress? What would be some prodev KPI’s?
+    * Graph: hours spent for prodev, per person
+    * Money invested in prodev, maybe per person?  Monthly?
+    * Participation numbers: what percentage of the company used at least 1 hour of prodev time? This could be tracked and reported on each quarter, with a goal of 100% participation.
```
Do we want people doing things quarterly? I could see someone spending a week between projects and having that be their annual usage

nerdstein commented on docs/03-policies/prodev.md:
```diff
@@ -0,0 +1,102 @@
+    * Participation numbers: what percentage of the company used at least 1 hour of prodev time? This could be tracked and reported on each quarter, with a goal of 100% participation.
+    * Would also be helpful to find historical participation percentage if the data exists to calculate it
+    * Monthly "state of prodev" report
+* How do we set prodev goals on both an individual and company-wide level?
```
I am floating the idea of service area leads helping with VSI for their respective area. Maybe prodev goals can be split up by area?

nerdstein commented on docs/03-policies/prodev.md:
```diff
@@ -0,0 +1,102 @@
+* Management regularly facilitating prodev trello board reviews - for most, this does not happen anymore. 
+    * Trello board templates for the prodev vision boards will still be available to those that find them useful
+* Previous confusion about “5% time” being shared between community participation and prodev
+    * The 5% time allocation is meant for community participation. While community participation and professional development are conceptually intertwined, the 5% time is meant to provide guidance for community participation only. The community participation page has been split off and moved [here](/community-participation.md)
```
Interesting. So, do we have any guidelines for projected or even desired pro dev use?

nerdstein commented on docs/03-policies/prodev.md:
```diff
@@ -0,0 +1,102 @@
+
+### Misc Stuff
+
+The target spreadsheet containing all prodev budget requests (fed in from the request form) is located [here](https://docs.google.com/spreadsheets/d/1vPhvHskomF7kBHvVw2WljTqel643SJpdNlyWgwKGaQE/edit#gid=1289927279)
```
I would make sure this is only available for CA. We may want to restrict access to the results if people do not want to share their expenses broadly

jmheaster committed:
Update prodev.md

Update of the hours budget section to include billability language

jmheaster committed:
Update prodev.md

Updated prodev sprint section

jmheaster committed:
Update prodev.md

Update to mentor section - add outside help text

jmheaster committed:
Update prodev.md

FAQ updates

jmheaster committed:
Update prodev.md

Appendix updates

jmheaster committed:
Update prodev.md

Fix square bracket issue

nerdstein reviewed (approved):
Approved
